en/docs/samples/oidc-token-encryption-sample.md

```diff
@@ -0,0 +1,196 @@
+
+    Leave these values as they are if you do not have any specific requirements.
+
+-----
+
+{!fragments/encrypt-id-tokens.md!}
+
+----
+
+You have successfully set up token encryption and the sample Playground application. You can now try out one of the grant type flows to recieve an encrypted ID token and decrypt it.
+
+- [Authorization grant flow](#try-authorization-grant-flow)
+- [Implicit grant flow](#try-implicit-grant-flow)
+- [Password grant flow](#try-password-grant-flow)
+
+----
+
+## Try authorization grant flow
+
+1.  Visit the URL <http://wso2is.local:8080/playground2/oauth2.jsp> to start the application.
+
+2.  Enter the following details and click **Authorize**.
+
+    -   **Authorization Grant Type:** Authorization Code
+
+    -   **Client ID:** (the client ID received when you registered the service provider)
+
+    -   **Scope:** openid
+
+    -   **Callback URL:** <http://wso2is.local:8080/playground2/oauth2client>
+
+    -   **Authorize Endpoint:** <https://localhost:9443/oauth2/authorize>
+
+3.  Sign in with user credentials.
+
+4.  Click **Approve** to consent to this action.
+
+5.  Provide the following details and click **Get Access Token**.  
+
+    -   **Callback URL:**  <http://wso2is.local:8080/playground2/oauth2client>
+
+    -   **Access Token Endpoint:** <https://localhost:9443/oauth2/token>
+
+    -   **Client Secret:** (the client secret received when you registered the service provider)  
+
+6.  At this point, the application receives the access token and the encrypted ID token.
+
+7.  In order to decrypt the ID Token, you need to provide the private key of the client.
+
+    1.  Import JKS into a PKCS12 formatted store.
+
+        ``` java
+        keytool -importkeystore -srckeystore testkeystore.jks -destkeystore testkeystore.p12 -srcstoretype JKS -deststoretype PKCS12 -srcstorepass wso2carbon -deststorepass wso2carbon -srcalias wso2carbon -destalias wso2carbon -srckeypass wso2carbon -destkeypass wso2carbon
+        ```
+
+    2.  Extract the private key into a file named `key.pem`.
+
+        ``` java
+        openssl pkcs12 -in testkeystore.p12 -out key.pem -passin pass:wso2carbon -passout pass:wso2carbon -nodes -nocerts
+        ```
+
+    3.  Open the created `key.pem` file using a text editor and you will see the extracted private key.
+
+    4.  Copy only the key string as shown in the sample below.
+
+        ![sample-key-string](../../assets/img/guides/sample-key-string.png)
+
+8.  Paste the copied private key in the **Client Private Key** text area.
+
+9.  Click **Decrypt** and the details of the decrypted ID Token will be displayed.
+
+## Try implicit grant flow
+
+1.  Visit the URL <http://wso2is.local:8080/playground2/oauth2.jsp> to
+    start the application.
+
+2.  Enter the following details and click **Authorize**.
+
+    -   **Authorization Grant Type:** Implicit
+
+    -   **Client ID:** (the client ID received when you registered the service provider)
```

Comment:
when listing the other fields we have given the exact value to enter...since this is a description and not the exact value, I put it in brackets to differentiate.